src/opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -555,7 +555,7 @@ inline void keywordR2R( SummaryConfig::keyword_list& /* list */,
 {
     const auto& location = keyword.location();
     std::string msg = "Region to region summary keyword: " + keyword.name() + " at " + location.filename + ", line " + std::to_string(location.lineno) + " is ignored";
-    parseContext.handleError(ParseContext::SUMMARY_UNHANDLED_KEYWORD, msg, errors);
+    // parseContext.handleError(ParseContext::SUMMARY_UNHANDLED_KEYWORD, msg, errors);
```

Comment:
I personally think this warning should be here and rather removed from opm-simulators. To actually provide a functionality to the user will always be a cooperation between opm-simulators and opm-common, in my opinion the warning about missing feature should be in the module which is "mostly to blame" for the missing functionality - or alternatively the module which needs to change the most to accomodate the missing feature.

For example tracer functionality will mostly (only ??) require changes in the opm-simulators code, and it makes sense to report the missing functionality there. When it comes to summary keywords the required changes will in most cases[1] be in opm-common and in my opinion it makes the most sense to report the warnings here.

[1]: This is not a hard rule - some summary keywords will require significant changes in opm-simulators, many will require some changes in opm-simulators, *but all new summary keywords will require changes in opm-common*.

Reply:
An additional argument for having the "Not support summary keywords" listed here is that it is less maintenance; here it is essentially automatic - the moment a new keyword is supported it will be picked up by a switch statement further up in the code - and * automatically removed* from the not supported list; in contrast the keyword list must be maintained manually in opm-simulators.

Instead of merging this change - I will make a PR in opm-simulators removing elements from the list there.

Reply:
Yes, keeping track of missing summary functionality may best belong in opm-common. However, those messages are very noisy at the moment and do not use the logging infrastructure.

Reply:
> However, those messages are very noisy at the moment as they do not use the logging infrastructure.

That is true - with #1946 the logging infrastructure will be used.

Reply:
Good, will give it a go.

Reply:
Observer that you also need opm-simulators#2810.

Reply:
Sure, I noticed. Compiling as we speak :-)